package io.github.libzeal.zeal.logic.evaluation.cause;

public class MaximumDepthExceededException extends RuntimeException{
}
